sort.c

```diff
@@ -220,7 +220,9 @@ static int compare_spam (const void *a, const void *b)
   {
     result = strcmp(aptr, bptr);
     if (result == 0)
+    {
       AUXSORT(result, a, b);
+    }
   }
 
```

Comment:
Someone changed the `AUXSORT()` macro.  It now contains multiple statements meaning this `if` doesn't do what it used to.